Lib/xml/etree/ElementInclude.py

```diff
@@ -106,12 +106,13 @@ def default_loader(href, parse, encoding=None):
 #     relative include file references.
 # @param max_depth The maximum number of recursive inclusions.
 #     Limited to reduce the risk of malicious content explosion.
-#     Pass a negative value to disable the limitation.
+#     Pass None to disable the limitation.
```

Comment:
Few lines above, specify that `elem` can be not only a root element, but an ElementTree instance.